tensorflow/core/common_runtime/gpu/gpu_id_manager_test.cc

```diff
@@ -30,30 +30,30 @@ PhysicalGpuId TfToPhysicalGpuId(TfGpuId tf) {
 
```

Comment:
Do we need to delete (or atleast update) the routine at lines 25-28? 
It does not seem like it is used at all, but has references `PhysicalGpuId` (guessing this test is not a part of CI, else would have expected this to trigger a compile failure)

Reply:
thanks. let me revise it.